src/components/top-bar/styles.ts

```diff
@@ -0,0 +1,87 @@
+import { css } from '@linaria/core'
+import { styled } from '@linaria/react'
+import { isDesktopOrBelow, isMobile, isTabletOrBelow } from '../../styles/media'
+import { ElButtonGroup } from '../button-group'
+import { ElNavSearchButton } from '../nav-search-button/styles'
+import { ElButtonNavIconItem } from '../nav-icon-item'
+
+export const elTopBarAvatar = css`
+  display: flex;
+  padding: var(--spacing-1, 4px) var(--spacing-none, 0px);
+  padding-left: var(--spacing-2, 8px);
+  align-items: center;
+  gap: 10px;
+`
+
+export const ElMainNavContainer = styled(ElButtonGroup)`
+  display: flex;
+  padding: var(--spacing-1, 4px) var(--spacing-none, 0px) var(--spacing-1, 4px) var(--spacing-6, 24px);
+  align-items: flex-start;
+  flex-grow: 1;
+`
+
+export const ElTopBarSecondaryNavContainer = styled(ElButtonGroup)`
+  flex-wrap: nowrap;
```

Comment:
this secondary nav naming has `TopBar` prefix because there is other component exported with the same name i.e under `layout/styles.tsx`, I think I will need to add prefix to the others too